src/app/zap-templates/zcl/data-model/chip/media-playback-cluster.xml

```diff
@@ -26,134 +26,84 @@ limitations under the License.
     <!-- TODO: Add feature map once it is supported -->
     <description>This cluster provides an interface for controlling Media Playback (PLAY, PAUSE, etc) on a media device such as a TV or Speaker.</description>
 
-    <attribute side="server" code="0x0000" define="MEDIA_PLAYBACK_STATE" type="ENUM8" min="0x00" max="0x02" writable="false" default="0x00" optional="true">playback state</attribute>
+    <attribute side="server" code="0x0000" define="MEDIA_PLAYBACK_STATE" type="PlaybackStateEnum" min="0x00" max="0x02" writable="false" default="0x00" optional="true">playback state</attribute>
```

Comment:
Again, the attribute names don't match the spec.  Followup.

Reply:
Added to issue: https://github.com/project-chip/connectedhomeip/issues/13213